lambda/shared/src/main/scala/feral/lambda/events/APIGatewayV2WebSocketEvent.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda.events
+
+import io.circe.Decoder
+
+sealed abstract class APIGatewayV2WebSocketEvent {
+  def stageVariables: Map[String, String]
+  def requestContext: WebSocketRequestContext
+  def body: String
```

Comment:
Looks like the body is optional, so we should use `Option[String]`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e40cab53d9971a3e0b34e8fb18ece74240d9425d/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L239